### ISSUE TITLE
Not using packed together jre

### ISSUE THREAD

**Issue by kukelekuuk00** [SquidHQ/launcher#4]
The Vanilla Minecraft launcher comes with a bundled together jre and uses that as a default. While SquidHQ uses the java installed on the computer. The difference here is that not every version of java is compatible. Some people get errors and can't even use the client because of this. The vanilla launcher uses 1.8.0_25.

Maybe look into Packr and bundle the required jre? https://github.com/libgdx/packr

**Comment by coelho** [comment]
Packr looks interesting! At the moment we use "launch4j", so this may be better.

**Comment by coelho** [comment]
Bad news is that Packr will not work as well as launch4j currently does.

Good news is that we kindof fixed this issue without even pushing a new Bootstrap:
https://github.com/SquidHQ/launcher/commit/729c1cf2601603d6988b8fc770dbffba2f09a539

Everything will auto-update as normal.

**Comment by khionu** [comment]
I'm a Moderator on the Official Technic Discord. We've seen plenty of people coming in with the Pixel Format error, and that version of Java is the one that works. It's why Mojang includes it.

Can thank Microsoft for screwing over Intel on this issue, it's particularly older (and semi older) Intel GPUs + Win 10.

I would recommend looking into a way to include it, yourselves.

**Comment by coelho** [comment]
@khionu The plan is to have the launcher download the JRE on Windows platforms, and then use the JRE it downloaded. This way we don't need to pack a JRE in the bootstrap executable, because the bootstrap/launcher has really no specific version requirement. Only MC does.

At the moment, it uses the JRE from the Minecraft launcher installation that is on your computer. See: https://github.com/SquidHQ/launcher/commit/729c1cf2601603d6988b8fc770dbffba2f09a539

This is pretty acceptable for now on the conversion process. Let me know if my plan makes sense?

**Comment by khionu** [comment]
Makes sense, but I strongly advise caution. Make sure you have the End User agree to the [License Agreement](http://www.oracle.com/technetwork/java/javase/terms/license/index.html), else you are liable.

**Comment by PhanaticD** [comment]
+1 on this, getting complaints of people being unable to use squid because they dont have java installed
